src/Settings.php

```diff
@@ -117,102 +125,202 @@ public function update( $settings ) {
+					),
+					'desc_tip'    => true,
+				),
+				'siwk_required_scopes_email'           => array(
+					'name'              => 'siwk_required_scopes',
+					'title'             => __( 'Required Customer Data', 'siwk' ),
+					'type'              => 'checkbox',
+					'label'             => __( 'Email Address', 'siwk' ),
+					'default'           => 'yes',
+					'disabled'          => true,
+					'custom_attributes' => array(
+						'checked' => 'true',
+					),
+				),
+				'siwk_required_scopes_name'            => array(
+					'name'              => 'siwk_required_scopes_name',
+					'type'              => 'checkbox',
+					'label'             => __( 'Full name', 'siwk' ),
+					'default'           => 'yes',
+					'disabled'          => true,
+					'class'             => 'siwk',
+					'custom_attributes' => array(
+						'checked' => 'true',
+					),
+				),
+				'siwk_required_scopes_phone'           => array(
+					'name'              => 'siwk_required_scopes_phone',
+					'type'              => 'checkbox',
+					'label'             => __( 'Phone number', 'siwk' ),
+					'default'           => 'yes',
+					'disabled'          => true,
+					'class'             => 'siwk',
+					'custom_attributes' => array(
+						'checked' => 'true',
+					),
+				),
+				'siwk_required_scopes_billing_address' => array(
+					'name'              => 'siwk_required_scopes_billing_address',
+					'type'              => 'checkbox',
+					'label'             => __( 'Billing address', 'siwk' ),
+					'description'       => __( 'These scopes are included by default, as necessary for creating a WooCommerce customer account in your shop.  More about available scopes with Sign in with Klarna <a href="https://docs.klarna.com/conversion-boosters/sign-in-with-klarna/integrate-sign-in-with-klarna/web-sdk-integration/#scopes-and-claims">here</a>. Additional scopes can be customized if applicable, more info <a href="https://gist.github.com/mntzrr/4bf23ca394109d40575f2abc05811ddc">here</a>.', 'siwk' ),
+					'disabled'          => true,
+					'class'             => 'siwk',
+					'custom_attributes' => array(
+						'checked' => 'true',
+					),
+				),
+				'siwk_optional_scopes_date_of_birth'   => array(
+					'name'    => 'siwk_optional_scopes_date_of_birth',
+					'title'   => 'Request Additional Customer Data',
+					'type'    => 'checkbox',
+					'label'   => __( 'Date of birth', 'siwk' ),
+					'default' => 'no',
+				),
+				'siwk_optional_scopes_country'         => array(
+					'name'    => 'siwk_optional_scopes_country',
+					'type'    => 'checkbox',
+					'label'   => __( 'Country', 'siwk' ),
+					'default' => 'no',
+					'class'   => 'siwk',
+				),
+				'siwk_optional_scopes_language'        => array(
+					'name'    => 'siwk_optional_scopes_language',
+					'type'    => 'checkbox',
+					'label'   => __( 'Language Preference', 'siwk' ),
+					'default' => 'no',
+					'class'   => 'siwk',
+				),
```

Comment:
Does not need to be changed for this, but could this be a multi-select instead of multiple checkboxes? Or do we thing that we need to have them as separate settings for some reason?